samples/client/petstore/dart-jaguar/openapi_proto/docs/UserApi.md

```diff
@@ -84,7 +84,7 @@ try {
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **body** | [**List&lt;User&gt;**](User.md)| List of user object | 
+ **body** | [**List&lt;User&gt;**](List.md)| List of user object | 
```

Comment:
not sure what append here as I didn't touch that... a regression @wing328 ?

Reply:
Looks like you're not using the latest master ...